softdev.bib

```diff
@@ -2807,3 +2807,45 @@ @misc{rebench18marr
     year = 2018,
     author = {Stefan Marr},
 }
+
+@misc{spidermonkey,
+    title = "{SpiderMonkey}",
+    howpublished = {\url{https://spidermonkey.dev}},
+    note = {Accessed: 2023-02-16},
+    year = 1995,
+    author = {Mozilla},
+}
+
+@misc{webkit,
+    title = "{WebKit}",
+    howpublished = {\url{https://webkit.org}},
+    note = {Accessed: 2023-02-16},
+    year = 1998,
+    author = {Apple},
+}
+
+@misc{pizlo17riptide,
+    title = "Introducing {Riptide}: {WebKit}’s retreating wavefront concurrent garbage collector",
+    author = "Filip Pizlo",
+    year = 2017,
+    month = Jan,
+    howpublished = {\url{https://webkit.org/blog/7122/introducing-riptide-webkits-retreating-wavefront-concurrent-garbage-collector/}},
+    note = {Accessed: 2023-02-16},
+}
+
+@misc{spidermonkeygc,
+    title = "{SpiderMonkey} garbage collector",
+    author = "Mozilla",
+    howpublished = {\url{https://firefox-source-docs.mozilla.org/js/gc.html}},
+    note = {Accessed: 2023-02-16},
+    year = 2018
+}
+
+@inproceedings{craciunas08compacting,
+    title = "A compacting real-time memory management system.",
```

Comment:
We don't want the `.`.

Reply:
Fixed: 2ce3a8b69d0205afb0457189ca3636d959a69898